.appveyor.yml

```diff
@@ -81,8 +81,8 @@ install:
   # we're disabling to get the initial AppVeyor to be green.
   #- set PATH=C:\Program Files\gs\gs9.20\bin;%PATH%
   # Install fig2dev.
-  - appveyor DownloadFile http://www.winfig.com/WinFIG-Dateien/WinFIG62.zip
-  - 7z x WinFIG62.zip > nul
+  - appveyor DownloadFile http://www.winfig.com/WinFIG-Dateien/WinFIG62.zip || true
+  - 7z x WinFIG62.zip > nul || true
```

Comment:
Thanks for fixing this.  It's worth a comment in the .yml file as to why we're doing this.  Did we not file an issue the last time this happened?  Maybe not -- we should file an issue and put a comment referencing it here.  (Ditto for doxygen.)